CHANGELOG.md

```diff
@@ -39,6 +39,8 @@ If you deploy with no distribution app your deployment will succeed but you may
 
 ### Added
 
+- **1012** - Adds `layers` config option to support deploying Lambdas with layers
+- **1098** - Added `useXRay` config option to enable AWS X-Ray for Lambdas.
```

Comment:
I guess this came from master but I prefixed my PR-only fix with 'PR' (i.e. `**PR1130**`) to make clear that it is from a pull request, and not a ticket (could be read as CUMULUS-1012/1098). Thoughts?